test/e2e/applicationset_test.go

```diff
@@ -915,7 +918,7 @@ func TestSyncPolicyCreateUpdate(t *testing.T) {
 
 		// Delete the ApplicationSet, and verify it deletes the Applications
```

Comment:
could you please change the comment, probably better explain why apps still exists

Reply:
Yeah, it is quite confusing here.

Reply:
Done

Reply:
@pasha-codefresh does this look good to you now?